### PR TITLE
Bugfix exception

### LINKED ISSUE #6: Errors
I tried to use the class FinancialsDownloader() class and didnt work :(.

Code example below:

```
fd = gm.FinancialsDownloader()
test = fd._download('APPL', 'is')
```
and gives the error:

with urllib2.urlopen(url) as response:
NameError: name 'urllib2' is not defined ...

In order to fix the problem I changed the line 300 
FROM `with urllib2.urlopen(url) as response:` TO` with urllib.request.urlopen(url) as response:`

As soon as I overcomed the problem above I am getting the following error that is caused by line **302**    `json_data = json.loads(json_text)` :

json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

I am on a dead-end situation now :(

### PR DESCRIPTION
**Pull request by linwoodc3**
Kept it super simple.  

The imports in my previous imports worked but I realized you didn't get a chance to completely review the changes from my last push in June. So, I reverted it back to the original.  

I pasted your original example in, and everything works fine.  

1. Bug fixes #6 and #7 
2.  Adds error handling for empty string and incorrect ticker
3. Includes unit tests folder if you ever implement CI

For what its worth, the CI tests would conduct automate tests to make sure you code works.  Therefore, you would know if all imports work before you reviewed a pull request. Every major library uses it; pandas, scikit-learn, etc.  And, the CI code doesn't impact anyone who uses your code. It only makes people who submit pull requests on github follow rules and not break your code.  

Hope this one is easier to follow.

### CODE REVIEW

**Comment by linwoodc3** [comment]
Here's the instructions on how to check a pull requests locally...to see if it works

https://help.github.com/articles/checking-out-pull-requests-locally/

File: good_morning/__init__.py [hunk 1]
```diff
@@ -24,12 +24,12 @@
...
 
-__name__ = 'morningstar'
+__name__ = 'good_morning'
 __author__ = 'Peter Cerno'
 __email__ = 'petercerno@gmail.com'
 __license__ = 'MIT.X11'
 __version__ = '0.1.0'
 __url__ = 'https://github.com/petercerno/good-morning'
-__description__ = 'Good Morning (morningstar) is a simple Python module for downloading fundamental financial data from [financials.morningstar.com](http://financials.morningstar.com/).'
+__description__ = 'Good Morning (good_morning) is a simple Python module for downloading fundamental financial data from [financials.good_morning.com](http://financials.good_morning.com/).'
```
Comment (petercerno) [inline good_morning/__init__.py]: Nit: http://financials.good_morning.com/ -> http://financials.morningstar.com/

File: good_morning/good_download.py [hunk 2]
```diff
@@ -18,14 +18,14 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 """Example showing how to download financial data from
-financials.morningstar.com for all tickers in S&P 500 (October 2015).
+financials.good_morning.com for all tickers in S&P 500 (October 2015).
```
Comment (petercerno) [inline good_morning/good_download.py]: Nit: financials.good_morning.com -> financials.morningstar.com

File: good_morning/good_morning.py [hunk 3]
```diff
@@ -17,24 +17,20 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
-"""Module for downloading financial data from financials.morningstar.com.
+"""Module for downloading financial data from financials.good_morning.com.
```
Comment (petercerno) [inline good_morning/good_morning.py]: Nit: financials.good_morning.com -> financials.morningstar.com

File: good_morning/good_morning.py [hunk 4]
```diff
@@ -17,24 +17,20 @@
...
-import pymysql
 import re
 import urllib.request
-from bs4 import BeautifulSoup, Tag
+from bs4 import BeautifulSoup
 from datetime import date
 
 class KeyRatiosDownloader(object):
-    u"""Downloads key ratios from http://financials.morningstar.com/
+    u"""Downloads key ratios from http://financials.good_morning.com/
```
Comment (petercerno) [inline good_morning/good_morning.py]: Nit: http://financials.good_morning.com/ -> http://financials.morningstar.com/

**Comment by petercerno** [comment]
I think that renaming `morningstar` to `good_morning` is much safer from legal perspective (as I am not sure if we are not violating some trademark policies when using `morningstar`). Also, using a different name allows to distinguish these two things in discussions.

> For what its worth, the CI tests would conduct automate tests to make sure you code works.

To be clear, I am not against CI. I think that having CI is a great idea, but I have two concerns:

- The code for enabling CI seems too complex to me. I do not understand it, and it might be also very confusing for other users. Moreover, it looks like it would require maintenance, which is an added cost.
- The main issue with `good_morning` is that it heavily depends on the data format returned by morningstar. If the data format changes, `good_morning` will break (or show wrong data). Therefore, it is necessary to test the consistency manually (from time to time).

**Pull request merged**